src/mvesuvio/analysis_reduction.py

```diff
@@ -995,24 +995,9 @@ def create_gamma_workspaces(self):
         profiles = self.calcGammaCorrectionProfiles(self._mean_widths, self._mean_intensity_ratios)
 
         # Approach below not currently suitable for current versions of Mantid, but will be in the future
```

Comment:
Is this comment still valid? The fix for VesuvioCalculateGammaBackground was merged end of 2023, so it should be available in current versions of Mantid.

Reply:
I am talking about the comment "# Approach below not currently suitable for current versions of Mantid, but will be in the future".

Reply:
Ah yes I forgot to remove it, thanks for spotting it!